web/src/components/ChatBotWidget.tsx

```diff
@@ -1,5 +1,11 @@
 import React, { ReactElement, useEffect } from 'react'
-import { useTheme } from 'styled-components'
+import styled, { useTheme } from 'styled-components'
+
+const ChatContainer = styled.div`
+  > div {
```

Comment:
i think this should be wrapped in a media query, since its only needed for tablet/desktop. for mobile it just replaces the icon
```js
  @media ${dimensions.mediumLargeViewport} {
    > div {
      right: calc(100% - 100vw + 33px);
    }
  }
  ```